### ISSUE TITLE
Create databases in RDS

### ISSUE THREAD

**Issue by jasonaowen** [PermanentOrg/notification-service#10]
Our non-local instances of the notifications service will use databases hosted by Amazon RDS. We need to create a database for each environment (dev, staging, and prod), and record the credentials.

The credentials will need to be added to the service definition we create in https://github.com/PermanentOrg/infrastructure/issues/45 as the `DATABASE_URL` environment variable; see also the [pull request where we define Sentry environment variables for upload-service](https://github.com/PermanentOrg/infrastructure/pull/11), and the documentation for [connection strings](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING).

**Comment by jasonaowen** [comment]
Last week, @mithuna and I paired on this task. We made some good progress, but did not finish; we plan on finishing this week.

We made RDS instances for dev and staging, and recorded credentials for the PostgreSQL superuser account in our password manager. Part of this work involved setting the appropriate security group, and configuring the security group appropriately, so that we could connect from the dev and staging backend servers.

We still need to create an RDS instance for production. We will also need to create a role in each cluster for the notification-service to connect as, since we should not use the superuser account, as well as a database for the service to use, instead of the default `postgres` database. Once we've created the role and database, we should have enough information to construct the `DATABASE_URL` we'll store as a secret in the infrastructure repo's configuration.
